word_language_model/README.md

```diff
@@ -35,8 +35,7 @@ optional arguments:
   --dropout DROPOUT     dropout applied to layers (0 = no dropout)
   --tied                tie the word embedding and softmax weights
   --seed SEED           random seed
-  --cuda                use CUDA
-  --mps                 enable GPU on macOS
+  --accel               activate support for an accelerator card
```

Comment:
To be consistent with other examples:
https://github.com/pytorch/examples/blob/a6d10a56b63ede465ff3239a2eb5eb00136b789e/fast_neural_style/neural_style/neural_style.py#L213-L214
I.e.:
```suggestion
  --accel               use accelerator
```